src/modules/dashboard/components/Community/Community.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { FC, useCallback, useState, useRef, useEffect } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { Address } from '~types/index';
+import {
+  useColonySubscribedUsersQuery,
+  useLoggedInUser,
+  useUserColonyAddressesQuery,
+  useSubscribeToColonyMutation,
+  cacheUpdates,
+} from '~data/index';
+import { domainsAndRolesFetcher } from '../../fetchers';
+import { getCommunityRoles } from '../../../transformers';
+import { useDataFetcher, useTransformer } from '~utils/hooks';
+import { ROLES_COMMUNITY } from '~constants';
+import { sortObjectsBy } from '~utils/arrays';
+
+import { Table, TableBody, TableCell } from '~core/Table';
+import { SpinnerLoader } from '~core/Preloaders';
+import UserListItem from '~admin/UserListItem';
+import Button from '~core/Button';
+
+import styles from './Community.css';
+
+const MSG = defineMessages({
+  loading: {
+    id: 'dashboard.Community.loading',
+    defaultMessage: "Loading Colony's users...",
+  },
+  callToSubscribe: {
+    id: 'dashboard.Community.callToSubscribe',
+    defaultMessage: `{noMembers, select,
+      true {No members yet. {action} to become the first member of this colony.}
+      other {{action} to become a member of this colony.}
+    }`,
+  },
+  subscribe: {
+    id: 'dashboard.Community.subscribe',
+    defaultMessage: `Subscribe now`,
+  },
+  subscribedReward: {
+    id: 'dashboard.Community.subscribedReward',
+    defaultMessage: `{star} Subscribed!`,
+  },
+});
+
+interface Props {
+  colonyAddress: Address;
+}
+
+enum ROLES {
+  FOUNDER = 'founder',
+  ADMIN = 'admin',
+  MEMBER = 'member',
+}
```

Comment:
It seems like we've been tending to prefer pascal case with enums elsewhere. Definitely a nit-pick, and I know we have a couple other instances that are uppercase (likely leftover from js -> ts migration). Totally your call here!